### ISSUE TITLE
[Bug] When OMV Instance is unreachable, this integration makes HA hang on bootup

### ISSUE THREAD

**Issue by d0rh4ns** [tomaae/homeassistant-openmediavault#132]
## Describe the issue

OMV Integration makes HA hang for far too long on bootup. Eventually there does seem to be a timeout but I'd argue it's far too long. Perhaps this could be configurable?

## How to reproduce the issue

1. Turn off/unplug from network your OMV instance
2. Restart HA
3. See how it's stuck at 'starting openmediavault' for a long time while other, perhaps more important stuff waits to be initialised

## Expected behavior

Have a shorter or configurable timeout

## Software versions

 - Home Assistant version: 
 - openmediavault integration version: 
 - openmediavault software version: 
HA: 2023.7.3
Integration version: 1.4.1

**Comment by tomaae** [comment]
Does that really hang HA, or you just see starting up message?

**Comment by d0rh4ns** [comment]
Ok, hang might not have been the correct word. It’s not like it’s unresponsive, but it doesn’t initiate other things that get initiated after OMV, which in some instances are important functions that I wouldn’t want to wait this long for if there’s a way around it

**Comment by MadWalnut** [comment]
@HannesK-ME Have you by any chance found a workaround for this? 

My OMV instance is a backup server that is almost always off. HA gets stuck for multiple minutes on `Starting OpenMediaVault. Not everything will be available until it is finished.`. Quite a few other integrations have to wait for this to time out before they show data. Wouldn't be such an issue if OMV would load last, but I don't think there is a way to configure the loading order.

**Comment by HannesK-ME** [comment]
@MadWalnut Nope, I just gave up on this since it wasn't super critical to me to have the OMV integration. I was also using this for a backup server that is almost always off, so I ran into this all the time. I thought HA was now initializing integrations in parallel though?! Back when I created this issue it would do them all in sequence, so nothing would get done until this timed out, which was extremely annoying. I'm surprised that there is still some sequencing taking place, but at least it should be far less severe now compared to 2023.

**Comment by MadWalnut** [comment]
@HannesK-ME Yeah, definitely something sequential still going on. It is especially noticeable for me since all my climate and light integrations seem to load after OMV so all dashboards are in a weird buggy state with almost all values missing.

@tomaae Can I help by providing any more details or testing something? This is the error in the log:

```
Logger: homeassistant.bootstrap
Source: bootstrap.py:999
First occurred: June 19, 2025 at 15:40:08 (4 occurrences)
Last logged: June 19, 2025 at 15:43:08

Waiting on integrations to complete setup: {('openmediavault', 'XXX'): XXX}
```

```
Logger: custom_components.openmediavault.omv_api
Source: custom_components/openmediavault/omv_api.py:107
integration: OpenMediaVault ([documentation](https://github.com/tomaae/homeassistant-openmediavault), [issues](https://github.com/tomaae/homeassistant-openmediavault/issues))
First occurred: June 19, 2025 at 15:41:29 (1 occurrence)
Last logged: June 19, 2025 at 15:41:29
OpenMediaVault hostname.com error while connect : unknown
```

Maybe it is worth checking out the implementation from the [Syncthing integration](https://github.com/home-assistant/core/tree/dev/homeassistant/components/syncthing), they seem to have some sort of handling for this scenario? Example from the log:

```
Logger: homeassistant.components.syncthing
Source: components/syncthing/__init__.py:156
integration: Syncthing ([documentation](https://www.home-assistant.io/integrations/syncthing), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+syncthing%22))
First occurred: June 19, 2025 at 18:44:05 (3 occurrences)
Last logged: June 19, 2025 at 23:44:17
The syncthing server 'https://syncthing.hostname.com' is not available. Sleeping 10 seconds and retrying
```

```
Logger: homeassistant.components.syncthing
Source: components/syncthing/__init__.py:127
integration: Syncthing ([documentation](https://www.home-assistant.io/integrations/syncthing), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+syncthing%22))
First occurred: June 19, 2025 at 19:44:06 (3 occurrences)
Last logged: 00:44:18
The syncthing server 'https://syncthing.hostname.com' is back online
```

**Comment by tomaae** [comment]
Not sure how much I can help since I dont use OMV for years. But I will try to update the integration once I finish with current one I'm reworking.

**Comment by MadWalnut** [comment]
Understandable, thank you for still taking the time. Feel free to ping me if there is a commit I can test for you.

**Comment by wolverinevn** [comment]
+1 reloading omv integration with omv on a high load machine hangs the hass frontend.
